apps/super-app/src/routes/LandingPage.tsx

```diff
@@ -0,0 +1,53 @@
+import './LandingPage.css';
+import { useEffect, useState } from 'react';
+import { Typography } from '@mui/material';
+import { Container } from '../common/components';
+import { ServiceCard } from '../common/components/ServiceCard';
+import {
+  getServiceCardAbouts,
+  getSupportDetailStrings,
+  ServiceCardAbout,
+  SupportDetailStrings,
+} from '../services/DetailServices';
+
+export function LandingPage() {
+  const [fixedStrings, setFixedStrings] = useState<SupportDetailStrings | null>(
+    null
+  );
+  useEffect(() => {
+    setFixedStrings(getSupportDetailStrings());
+  }, []);
+  if (!fixedStrings) {
+    // TODO return something (hint or loading spinner) since this is landing page, we can't return null
```

Comment:
IMO returning `null` is ok, since it'll just look like the page is loading. But a spinner is nicer, yes :)

Reply:
Let's keep it as TODO